src/main/java/entity/post/Post.java

```diff
@@ -18,18 +21,6 @@ public interface Post {
      */
     String getContent();
 
```

Comment:
Do we differentiate between different posts implicitly by using their index in the list in the club object? Slightly confused.

Reply:
Since we settled on all posts having a title and text content, we would not need to differentiate them in terms of type.
For clubs, each individual club has an array of posts. The most recent post being at the end of the list, and the oldest being the first element of the list. Moreover, all posts have a time stamp. I hope this somewhat answers the question... I do not fully understand what you mean, feel free to ask more if you are still confused

Reply:
It was a misunderstanding on my part, I think I've made sense of it now!

Reply:
Great! I will resolve the conversation then. Let me know on discord if you still have any questions in the future.